lib/tasks/redirect_legacy_email_signups.rake

```diff
@@ -0,0 +1,31 @@
+desc "Redirect legacy email signups"
+task redirect_legacy_email_signups: :environment do
+  specialist_sectors_info = Topic.where(state: %w[draft published]).where.not(parent_id: nil).map do |ss|
+    { state: ss.state, slug: ss.full_slug }
+  end
+
+  specialist_sectors_info.map do |specialist_sector|
+    base_path = "/topic/#{specialist_sector[:slug]}/email-signup"
+    destination = "/topic/#{specialist_sector[:slug]}"
+
+    redirect = {
+      base_path: base_path,
+      document_type: "redirect",
+      schema_name: "redirect",
+      publishing_app: "collections-publisher",
+      update_type: "major",
+      redirects: [
+        { path: base_path, type: "exact", destination: destination },
+      ],
+    }
+
+    content_id = SecureRandom.uuid
+    Services.publishing_api.put_content(content_id, redirect)
+
+    if specialist_sector[:state] == "published"
+      Services.publishing_api.publish(content_id, "republish")
```

Comment:
I don't think we need the "republish" argument here, these are technically new publishing of the redirect document and not a republishing of it.